app/grandchallenge/evaluation/emails.py

```diff
@@ -42,43 +50,83 @@ def send_successful_evaluation_email(evaluation):
     challenge = evaluation.submission.phase.challenge
 
     recipients = list(challenge.get_admins())
-    message = (
-        f"Dear {{}},\n\n"
-        f"There is a new result for {challenge.short_name} from "
-        f"{evaluation.submission.creator.username}. "
-    )
+
     if evaluation.published:
-        leaderboard_url = reverse(
-            "evaluation:leaderboard",
-            kwargs={
-                "challenge_short_name": challenge.short_name,
-                "slug": evaluation.submission.phase.slug,
-            },
+        recipients.append(evaluation.submission.creator)
+
+    for recipient in recipients:
+        message = format_html(
+            "Dear {recipient},\n\n"
+            "There is a new result for {challenge} from {user}.",
+            recipient=recipient.username,
+            challenge=challenge.short_name,
+            user=evaluation.submission.creator.username,
         )
-        message += (
-            f"You can view the result on the leaderboard here: "
-            f"{leaderboard_url}"
+        if evaluation.published:
+            leaderboard_url = reverse(
+                "evaluation:leaderboard",
+                kwargs={
+                    "challenge_short_name": challenge.short_name,
+                    "slug": evaluation.submission.phase.slug,
+                },
+            )
+            message += format_html(
+                "You can view the result on the leaderboard here: {leaderboard}\n\n",
+                leaderboard=leaderboard_url,
+            )
+        else:
+            message += format_html(
+                "You can publish the result on the leaderboard here: {leaderboard}\n\n",
+                leaderboard=evaluation.get_absolute_url(),
+            )
+        message += format_html(
+            "Regards,\n"
+            "{site}\n\n"
+            "This is an automated service email from {domain}",
+            site=site.name,
+            domain=site.domain,
         )
-        recipients.append(evaluation.submission.creator)
-    else:
-        message += (
-            f"You can publish the result on the leaderboard here: "
-            f"{evaluation.get_absolute_url()}"
+        send_mail(
+            subject=(
+                f"[{site.domain.lower()}] "
+                f"[{challenge.short_name.lower()}] "
+                f"New Result"
+            ),
+            message=message,
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            recipient_list=[recipient.email],
         )
-    message += (
-        f"\n\n"
-        f"Regards,\n"
-        f"{site.name}\n\n"
-        f"This is an automated service email from {site.domain}."
-    )
+
+
+def send_missing_method_email(evaluation):
```

Comment:
In `models.py` you send a `submission` to this function, not an `evaluation`. Would you be able to update the variable name and usage in this function?